store/sqlstore/upgrade.go

```diff
@@ -968,7 +968,8 @@ func upgradeDatabaseToVersion533(sqlStore *SqlStore) {
 }
 
 func upgradeDatabaseToVersion534(sqlStore *SqlStore) {
-	// if shouldPerformUpgrade(sqlStore, Version5330, Version5340) {
-	// 	saveSchemaVersion(sqlStore, Version5340)
-	// }
+	//if shouldPerformUpgrade(sqlStore, Version5330, Version5340) {
+	sqlStore.AlterColumnTypeIfExists("Roles", "Permissions", "longtext", "text")
+	//	saveSchemaVersion(sqlStore, Version5340)
+	//}
```

Comment:
Should we just remove the commented code?

Reply:
@BenCookie95 
This commented code is done on purpose. They will uncomment it when server team upgrades servers to 5.34. We won't want to remove it.